leveldbstore/store.go

```diff
@@ -0,0 +1,48 @@
+package leveldbstore
+
+import (
+	"github.com/eventials/go-tus"
+	"github.com/mitchellh/go-homedir"
+	"github.com/syndtr/goleveldb/leveldb"
+	"path/filepath"
+)
+
+type LeveldbStore struct {
+	db *leveldb.DB
+}
+
+func NewLeveldbStore() (tus.Store, error) {
+	home, err := homedir.Dir()
```

Comment:
Just personal preference. There was no set precedence in the codebase yet on how to handle this. If you prefer a path I can change that.